site/.dumirc.ts

```diff
@@ -217,9 +217,9 @@ export default defineConfig({
             order: 2,
         },
         {
-            slug: 'chart-advisor',
+            slug: 'advisor',
             title: {
-                zh: '图表推荐（ChartAdvisor）',
+                zh: '图表推荐（Advisor）',
```

Comment:
@LAI-X and u both modify `site`?

Reply:
that's reasonable，because she don't know about rename matters.